api/controllers/Cid_EntryController.js

```diff
@@ -26,6 +26,11 @@ module.exports = {
       }
     ),
 
+  prebookingProcess: movements =>
```

Comment:
not too sure I agree, as the others do have an explicit function: update / create the item, however they have all been given the same implicit name {collection}Process() ... but I have made the change.